src/webrtc/plugin/audiobridge-plugin.js

```diff
@@ -101,7 +116,26 @@ AudiobridgePlugin.prototype.changeRoom = function(roomId, options) {
     request: 'changeroom',
     room: roomId
   }, options);
-  return this.sendWithTransaction({body: body});
+  return this.sendWithTransaction({body: body})
+    .then(function() {
+      this._currentRoomId = roomId;
+    }.bind(this));
+};
+
+/**
+ * @param {int} roomId
+ * @param {Object} [options]
+ * @param {int} [options.id]
+ * @param {string} [options.display]
+ * @param {boolean} [options.muted]
+ * @param {int} [options.quality]
+ * @return {Promise}
+ */
+AudiobridgePlugin.prototype.connectRoom = function(roomId, options) {
+  if (this._currentRoomId) {
```

Comment:
should we check if `roomId == this._currentRoomId` and resolve directly in this case?